.tool-versions

```diff
@@ -0,0 +1,2 @@
+nodejs 24.4.0
```

Comment:
What does this do?

Reply:
It's a way to specify the version of the tools we are using,
it's like `.ruby-version` , but more generic
we use it in CDA,
it is used by `asdf` and other versioning tools